app/models/filepath.rb

```diff
@@ -24,6 +24,16 @@ def self.offenders
              select("STRING_AGG(DISTINCT(cve),', ') AS cves")
   end
 
+  def vulnerabilities(id)
```

Comment:
```suggestion
  def vulnerabilities
```
Per my note above - this should still work because `id` is a local method on the Filepath model that returns the current ID.